src/resources/track/track.service.ts

```
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { CreateTrackDto } from './dto/create-track.dto';
+import { UpdateTrackDto } from './dto/update-track.dto';
+
+@Injectable()
+export class TrackService {
+  create(createTrackDto: CreateTrackDto) {
```

ESLint

[error] 'createTrackDto' is defined but never used.

(@typescript-eslint/no-unused-vars)